src/transformers/image_processing_utils_fast.py

```diff
@@ -304,9 +304,9 @@ def compile_friendly_resize(
         A wrapper around `F.resize` so that it is compatible with torch.compile when the image is a uint8 tensor.
         """
         if image.dtype == torch.uint8:
-            image = image.float() / 256
+            image = image.float() / 255
             image = F.resize(image, new_size, interpolation=interpolation, antialias=antialias)
-            image = image * 256
+            image = image * 255
             image = torch.where(image > 255, 255, image)
             image = torch.where(image < 0, 0, image)
```

Comment:
btw, it might be more optimal to use `torch.clamp(image, 0, 255)` once instead of `torch.where` twice

Reply:
Normally I would agree! But this is on purpose : https://github.com/huggingface/transformers/pull/38540

Reply:
Ok, according to the PR it seems we have to revert this PR to use 256 and keep torch.where. To prevent this code from further regression we have to either add tests that fails on CI (cuda) if modified or **properly** comment the code